app/(Components)/(Form)/BotzEvent/VideoIntro.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {Input} from '../Input';
+import {InputDate} from '../InputDate';
+import {InputTime} from '../InputTime';
+
+export const VideoIntro: React.FC<{introInputs: object}> = ({introInputs}) => {
+	return (
+		<section>
+			{Object.entries(introInputs).map(([label, input], key) => {
+				const InputComponent =
+					label === 'date' ? InputDate : label === 'time' ? InputTime : Input;
```

Comment:
Change this condition 😉 Try to avoid interweaving ternaries for better readability !

Reply:
Done 👍